docusaurus/docs/guides/wildcard-dns-kerberos.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Wildcard DNS and Kerberos
+sidebar_label: Wildcard DNS
+sidebar_position: 50
+---
+
+### How to Tunnel Kerberos/Active Directory DNS Queries
+
+Kerberos/AD domain members can communicate with their domain controller (DC) using a Ziti network. This involves forwarding DNS queries to the DC's nameservers with a Ziti service.
```

Comment:
This doesn't feel very 'guide-like' to me. it's providing me a bunch of ancillary information and not "here is how you do it" which is what I would expect to see from a guide. Do this do that etc. It's not feeling like 'a guide' to me